flexmeasures/data/models/planning/storage.py

```diff
@@ -233,6 +233,35 @@ def _prepare(self, skip_validation: bool = False) -> tuple:  # noqa: C901
                 max_value=convert_units(power_capacity_in_mw, "MW", sensor.unit),
             )
 
+        soc_gain = self.flex_model.get("soc_gain", [])
+        soc_usage = self.flex_model.get("soc_usage", [])
+
+        all_stock_delta = []
+
+        for is_usage, soc_delta in zip([False, True], [soc_gain, soc_usage]):
+            for component in soc_delta:
+                stock_delta_series = get_continuous_series_sensor_or_quantity(
+                    quantity_or_sensor=component,
+                    actuator=sensor,
+                    unit="MW",
+                    query_window=(start, end),
+                    resolution=resolution,
+                    beliefs_before=belief_time,
+                )
+
+                stock_delta_series *= resolution / timedelta(hours=1)  # MW -> MW
```

Comment:
I guess the comment should read `# MW-> MWh`, or even: `# MW -> MWh / resolution`

Example: 4 MW sustained over 15 minutes gives 1 MWh.

Reply:
True, good catch!